src/styles/main.scss

```diff
@@ -1,3 +1,226 @@
+/* #region Normalize */
 body {
+  font-family: "Roboto", Helvetica, sans-serif;
   margin: 0;
+  box-sizing: border-box;
+}
+/* #endregion */ 
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-transform: uppercase;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  text-decoration: none;
+  color: #060b35;
+  height: 60px;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  transition-duration: 0.5s;
+
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  position: absolute;
+  content: "";
+  display: block;
+  height: 4px;
+  background-color: #00acdc;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 3px 41px 73px;
+  max-width: 1082px;
+  min-width: 1082px;
+
+  &__container {
+    margin-right: 48px;
+    margin-top: 47px;
+ 
+  }
+
+  &__container:nth-child(4n) {
```

Comment:
use `card `class here
`    card:nth-child(4n) {
      margin-right: 0;
    }`
do not use `clear:right;` 
so every 4th card will have 0 margin on its right

Reply:
&:nth-child(4n) {
    margin-right: 0;